client/src/helpers/APICalls/changeSitterStatus.ts

```diff
@@ -0,0 +1,21 @@
+import { AuthApiData } from '../../interface/AuthApiData';
+
+interface PatchFetchOptions {
+  method: string;
+  credentials: RequestCredentials;
+}
+
+const changeSitterStatus = async (_id: string): Promise<AuthApiData> => {
+  const fetchOptions: PatchFetchOptions = {
+    method: 'PATCH',
+    credentials: 'include',
+  };
+
+  return await fetch(`/profile/sitter?` + new URLSearchParams({ _id }), fetchOptions)
```

Comment:
```suggestion
  return await fetch(`/profile/sitter` + new URLSearchParams({ _id }), fetchOptions)
```

Reply:
This change causes the following error: 
<img width="1439" alt="Screen Shot 2021-12-07 at 04 23 22" src="https://user-images.githubusercontent.com/18492009/145012256-6d166965-8686-4b8b-bb31-0d663e614efe.png">

Reply:
I fixed the editProfile controller, and now fetch uses a post method to send data to the editProfile controller. So this change has been resolved.